content/fast/adaptive-serving-based-on-network-quality/index.md

```diff
@@ -0,0 +1,103 @@
+---
+page_type: guide
+title: Adaptive serving based on network quality
+author: mihajlija
+description: |
+  Using Network Information API to adapt which assets you're serving to users based on the quality of their connection.
+web_lighthouse: N/A
+web_updated_on: 2019-04-01
+web_published_on: 2019-04-01
+wf_blink_components: N/A
+---
+
+# Adaptive serving based on network quality
+
+Loading a website can be a very different experience depending on the network conditions. Everything is usually smooth when you are on a fast network, but when you are on the go with a humble data plan and spotty connection, or stuck with a laptop on a sluggish coffee shop wifi, it's a different story.
+
+One way to deal with this is by adapting which assets you're serving to users based on the quality of their connection. This is now [possible](https://caniuse.com/#feat=netinfo) with the [Network Information API](https://developer.mozilla.org/en-US/docs/Web/API/Network_Information_API) which enables web applications to access information about the user's network.
+
+## Usage
+
+There are many ways you can use this network information to improve the user experience:
+
+* Switch between serving high-definition and low-definition content based on the user's network. 
+* Decide whether to preload resources. 
+* Defer uploads and downloads when users are on a slow connection. 
+* Enable offline mode if the network quality is not good enough to load the app and use the features.
+* Warn users that doing something (e.g. watching video) over cellular could cost them money.
+* Use it in your analytics to gather data on your users’ network quality.
+
+Many applications are already doing something similar. For example, YouTube, Netflix and most other video (or video calling) services automatically adjust the resolution during streaming. When Gmail is loading, it provides users with a link to "load basic HTML (for slow connections)" - a simplified (yet functional) version of Gmail. 
+
+<img class="screenshot" src="./loading_gmail_slow_connection.png" alt="A link to load basic HTML version of Gmail when users are on slow connections">
+
+With the Network Information API you can now implement these functionalities in your own applications without much effort.
```

Comment:
Suggestion to remove this sentence, 'With the'. It's not doing any extra work, especially since you aren't showing how to do it here.